From 4124bacb47eda6fba813ab5fa735291010c1ad36 Mon Sep 17 00:00:00 2001
From: Ken Ma <make@marvell.com>
Date: Tue, 14 Nov 2017 15:28:33 +0800
Subject: [PATCH 1244/1345] arm64: dts: a80x0: disable ihb1 AXI monitor by
 default

commit  8ef8cd9cbc2fe453aca0446b7e4700fc9775f94a from
https://github.com/MarvellEmbeddedProcessors/linux-marvell.git

The AXI monitors are only used for debug purpose,
thus this patch disables ihb1 AXI monitors by default.
Whenever the debug is needed, the user can enable the
AXI monitor on demand.

Change-Id: Icb372fe7878b605659a9024d3474046da514fe1a
Signed-off-by: Ken Ma <make@marvell.com>
Reviewed-on: http://vgitil04.il.marvell.com:8080/46391
Reviewed-by: Victor Gu <xigu@marvell.com>
Tested-by: iSoC Platform CI <ykjenk@marvell.com>
Reviewed-by: Nadav Haklai <nadavh@marvell.com>
Reviewed-by: Hanna Hawa <hannah@marvell.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm64/boot/dts/marvell/armada-8020.dtsi |   12 ++++++++++++
 arch/arm64/boot/dts/marvell/armada-8040.dtsi |   12 ++++++++++++
 2 files changed, 24 insertions(+)

diff --git a/arch/arm64/boot/dts/marvell/armada-8020.dtsi b/arch/arm64/boot/dts/marvell/armada-8020.dtsi
index deeecdf..39fc88a 100644
--- a/arch/arm64/boot/dts/marvell/armada-8020.dtsi
+++ b/arch/arm64/boot/dts/marvell/armada-8020.dtsi
@@ -149,6 +149,12 @@
 				#stream-id-cells = <1>;
 			};
 		};
+		axim-hb1-rd@3c8000 {
+			status = "disabled";
+		};
+		axim-hb1-wr@3c9000 {
+			status = "disabled";
+		};
 		pcie@0x600000 {
 			msi-parent = <&gic_v2m0>;
 		};
@@ -195,6 +201,12 @@
 				status = "okay";
 			};
 		};
+		axim-hb1-rd@3c8000 {
+			status = "disabled";
+		};
+		axim-hb1-wr@3c9000 {
+			status = "disabled";
+		};
 		pcie@0x600000 {
 			msi-parent = <&gic_v2m0>;
 		};
diff --git a/arch/arm64/boot/dts/marvell/armada-8040.dtsi b/arch/arm64/boot/dts/marvell/armada-8040.dtsi
index 94a9589..2b01a58 100644
--- a/arch/arm64/boot/dts/marvell/armada-8040.dtsi
+++ b/arch/arm64/boot/dts/marvell/armada-8040.dtsi
@@ -162,6 +162,12 @@
 				#stream-id-cells = <1>;
 			};
 		};
+		axim-hb1-rd@3c8000 {
+			status = "disabled";
+		};
+		axim-hb1-wr@3c9000 {
+			status = "disabled";
+		};
 		pcie@0x600000 {
 			msi-parent = <&gic_v2m0>;
 		};
@@ -208,6 +214,12 @@
 				status = "okay";
 			};
 		};
+		axim-hb1-rd@3c8000 {
+			status = "disabled";
+		};
+		axim-hb1-wr@3c9000 {
+			status = "disabled";
+		};
 		pcie@0x600000 {
 			msi-parent = <&gic_v2m0>;
 		};
-- 
1.7.9.5

